### DISCUSSION TITLE
Scheduled reminders: Please add exclude repository list

### DISCUSSION THREAD

**Discussion by plynch-chwy** [community/community#57442] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Problem:
Our team utilizes template repos to start off a lot of projects, so we have selected to track "All repositories" for PR reminders. For better or worse, there are a couple repos in our org which utilizes the CODEOWNERS to give teams access to certain paths in the repo and they build up a lot of PRs.

We do not want those PRs listed, but the only way to ignore things with the current reminders is "Ignored terms" and "Ignored labels". Unfortunately there is some overlap with those org wide repos and the repos our team uses, so neither of those ignore options are helpful for us. The other option we could use today is make it a best practice to manually add any new repo to the tracked repos list, but that is tedious since only the team maintainers have access to update the reminder.

Possible solution:
One preferred option is to have a list of repos to ignore. That would let us add the few "problem" repos to the list while allowing us to continue with tracking all new repos the team creates.

**Comment by mkaniaa** [comment]
I would love such a feature! I am assigned to a number of repositories that I am no longer involved with, but because I participated in their development process, I am still on the CODEOWNERS review list. It would be great to be able to ignore reminders from these repositories!
